src/main/java/com/echobox/api/linkedin/client/DefaultLinkedInClient.java

```diff
@@ -624,6 +624,7 @@ protected Response makeRequestAndProcessResponse(Requestor requestor) {
         && HttpURLConnection.HTTP_FORBIDDEN != response.getStatusCode()
         && HttpURLConnection.HTTP_NOT_MODIFIED != response.getStatusCode()
         && HttpURLConnection.HTTP_GATEWAY_TIMEOUT != response.getStatusCode()
+        && 422 != response.getStatusCode()
```

Comment:
```suggestion
        && HttpStatus.SC_UNPROCESSABLE_ENTITY != response.getStatusCode()
```
If `HttpURLConnection `doesn't provide constant for `422`, maybe we can use `HttpStatus`? could replace `429 `as well.

Reply:
I agree, will make that change - I made it for the whole class.